html/inc/boinc_db.inc

```diff
@@ -303,6 +303,10 @@ class BoincTeamDelta {
         $db = BoincDb::get();
         return $db->enum('team_delta', 'BoincTeamDelta', $where_clause);
     }
+    static function delete_for_user($user_id) {
+        $db = BoincDb::get();
+        return $db->delete_aux('team_delta', "userid=$user_id");
```

Comment:
There should be an index on `team_delta.user_id` to make this query efficient.

Reply:
added